src/adapter/src/coord.rs

```diff
@@ -5491,6 +5605,21 @@ impl<S: Append + 'static> Coordinator<S> {
             }
         }
 
+        self.remove_global_read_holds_storage(
+            sources_to_drop
+                .iter()
+                .chain(tables_to_drop.iter())
+                .chain(recorded_views_to_drop.iter().map(|(_, id)| id))
```

Comment:
recorded views needs to be in both?

Reply:
Yeah if you take a look at `sequence_create_recorded_view` there's a call to `initialize_storage_read_policies` and `ship_dataflow`, which calls `initialize_compute_read_policies`. So the recorded view gets a storage and compute read policy.

@teskje can you confirm that the storage and compute objects for recorded views have the same `GlobalId`? I based this on `drop_recorded_views` which seems to use the same ID to drop the sink in compute and the source in storage.

Reply:
Yes, that's correct!